main.py

```
@@ -214,6 +214,86 @@ def on_window_loaded(window):
+    """Launch the auto-updater with enhanced search and error handling"""
+    import subprocess
+    
+    def show_windows_error_dialog(title, message):
+        """Display a native Windows error dialog using ctypes"""
+        if platform.system() == "Windows" and ctypes:
+            try:
+                MB_ICONERROR = 0x00000010
+                MB_OK = 0x00000000
+                
+                # Create message box
+                result = ctypes.windll.user32.MessageBoxW(
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)